obs-package-status/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM opensuse/tumbleweed
```

Comment:
nitpick: I don't know, I would switch to a base OS that has a long lived security policy. Like Leap. What do you think?

Reply:
why? we normally backport security updates from Factory. So if there is one base OS which has all security updates, should be Tumbleweed, no?

Reply:
>  we normally backport security updates from Factory.

Not sure this is true, but if you say so :smile: